src/App.js

```diff
@@ -3,16 +3,19 @@ import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
 import Home from './components/Home';
 import Navigation from './components/Navigation';
 import Categories from './components/Categories';
+import './app.css';
 
 function App() {
   return (
-    <Router>
-      <Navigation />
-      <Routes>
-        <Route path="/" element={<Home />} />
-        <Route path="/categories" element={<Categories />} />
-      </Routes>
-    </Router>
+    <div className="App">
+      <Router>
+        <Navigation />
+        <Routes>
+          <Route path="/" element={<Home />} />
+          <Route path="/categories" element={<Categories />} />
+        </Routes>
```

Comment:
Like in my previous comment, I suggest that you try to make your design look as close to the design template as possible. In the template design, there is a horizontal line separating the booklist from the add-book-form (screenshot below) and I suggest that you add this in your design to make it look just like the design template. Inside your ```<Home />``` component (line 14) you can put a horizontal line, ```<hr/>```, between the ```<BookList />``` component and the ```<AddBooks addBooks={add} />``` component and give it padding on the left and on the right: 
![image](https://user-images.githubusercontent.com/107795045/232146367-8bc8bf02-20c3-4299-bea9-4474155d4e54.png)

Your design: 
![image](https://user-images.githubusercontent.com/107795045/232146516-0d205368-2473-4881-87ba-fc109ae55e1f.png)

Template design: 
![image](https://user-images.githubusercontent.com/107795045/232146758-5f1b4d67-cf82-4cca-bd76-2fd1a2f4fbaa.png)